packages/jmix-front-generator/src/generators/react-typescript/addon/answers.ts

```diff
@@ -0,0 +1,17 @@
+
+import {StudioProjectInfo} from '../../../common/studio/studio-integration';
+import {ProjectModel} from "../../../common/model/cuba-model";
+import {YeomanGenerator} from "../../../building-blocks/YeomanGenerator";
+import {CommonGenerationOptions} from "../../../common/cli-options";
+import {StudioTemplateProperty} from "../../../common/studio/studio-model";
+
+export interface Answers {}
+
+export const allQuestions: StudioTemplateProperty[] = [];
+
+export const getAnswersFromPrompt = async (
+    projectModel: ProjectModel, gen: YeomanGenerator, options: CommonGenerationOptions
+  ): Promise<Answers> => {
+ 
+    return {} as Answers;
+}
```

Comment:
We should probably modify `defaultGetAnswersFromPrompt` so that it returns empty answers when there are no questions.